AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -1273,6 +1273,7 @@ beasoku.com##.l-rss-header
 !
 !   SECTION: Widgets - Marketing (specific)
 !
+global.honda##.p-post22__caption
```

Comment:
There is a more obvious class name (`sns` part frequently used for social sharing buttons)
![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/a0066ffd-cd2e-47c4-896c-8749d8949bc5)


Also wrong filter - please move to the Social filter => specific.txt

Reply:
@Alex-302 fixed